BaseClasses.py

```diff
@@ -957,6 +942,18 @@ def can_reach_private(self, state: CollectionState) -> bool:
                 return True
         return False
 
+    @property
+    def hint_text(self) -> str:
+        if self.hint:
+            return self.hint
+        return self.name
+
+    @property
+    def get_entrance(self) -> Entrance:
```

Comment:
Excuse me if I'm wrong, but shouldn't property names not have verbs in them? It's implied as a getter, right?

Reply:
I couldn't find this mentioned specifically in PEP8 but you are correct so I've changed it.